conda_requirements.txt

```diff
@@ -0,0 +1,9 @@
+tokenizers>=0.11.1,!=0.11.3,<0.13
+allennlp
```

Comment:
I think this needs to be pinned right? The recent changes made to main since 2.0.0 introduced compatibility with allennlp 2.10 but that also breaks compatibility with older versions of allennlp. We should probably try to find out the minimum version this is compatible with and add that requirement here.

Reply:
Sure. I will dig out. Just wondering why the minimum compatible version should be pinned. Don't we want to use the latest version?

Reply:
Yes, agreed!

Reply:
I think @ksteimel means that now that we have refactored the code, gector will no longer work with old versions of AllenNLP so we want to have a lower bound here, we want the upper bound to stay unpinned so we will always use the latest version.

Reply:
Thanks for explaining!